<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMockWeight</class>
 <widget class="QWidget" name="FormMockWeight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weight</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>轴一重</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="wt1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>轴二重</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="wt2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>轴三重</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="wt3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>轴四重</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="wt4"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>轴五重</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="wt5"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>车速</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="speed"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>轴六重</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="wt6"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>轴七重</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="wt7"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>轴八重</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="wt8"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>轴九重</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="wt9"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>轴十重</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="wt10"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_allnumb">
       <property name="text">
        <string>累计车数：0000000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_axle"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>车型</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_type"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>车轴组数</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_group"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>车轴数</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QTextEdit" name="pingjun"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget">
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>车型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>车速</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴一重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴二重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴三重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴三重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴四重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴五重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴六重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴七重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>轴八重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="push_back">
       <property name="text">
        <string>倒车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_clear">
       <property name="text">
        <string>清空缓存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushRand">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_send">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
